### PR TITLE
add contains and toString

### PR DESCRIPTION
**Pull request by calvernaz**
Adds toString and contains methods. These two have been overridden in community and will be removed once this is merged

### CODE REVIEW

**Review by whummer — COMMENTED** [review]
Nice! Thanks for adding this @calvernaz , and for adding the snapshot test. 🎉 

Looks like there's only a small discrepancy - I guess we could create a custom serializer to convert `dict` objects to string.. (VTL uses the Java notation of Map.toString)
```
>> match key: render-result-1
	(~) / '{k1=v1, k2=v2}' → "{'k1': 'v1', 'k2': 'v2'}" ... (expected → actual)
```

**Comment by calvernaz** [comment]
/

> Nice! Thanks for adding this @calvernaz , and for adding the snapshot test. 🎉
> 
> Looks like there's only a small discrepancy - I guess we could create a custom serializer to convert `dict` objects to string.. (VTL uses the Java notation of Map.toString)
> 
> ```
> >> match key: render-result-1
> 	(~) / '{k1=v1, k2=v2}' → "{'k1': 'v1', 'k2': 'v2'}" ... (expected → actual)
> ```

good catch, have a fix for it, hopefully is good enough

File: airspeed/operators.py [hunk 1]
```diff
@@ -27,11 +27,16 @@ def dict_put(self, key, value):
     return existing
 
 
+def dict_to_string(self):
+    return "{" + ", ".join([f"{k}={v}" for k, v in self.items()]) + "}"
```
Comment (whummer) [inline airspeed/operators.py:31]:
```suggestion
def dict_to_string(obj: dict) -> str:
    return "{" + ", ".join([f"{k}={v}" for k, v in obj.items()]) + "}"
```

**Review by whummer — APPROVED** [review]
LGTM 👍 (only a tiny nit suggestion..)

Can you please bump the version and update the change log in the README, then we can release a new version right away.. 👍

**Pull request merged**